utility/doc/PROTOCOL_SESSION.md

```diff
@@ -1,81 +1,54 @@
-## Protocols
-
-### Session Protocol
-
-`Pocket` implements the V1 Utility Specification's Session Protocol by satisfying the following interface:
-
-```golang
-type Session interface {
-    NewSession(sessionHeight int64, blockHash string, geoZone GeoZone, relayChain RelayChain, application *coreTypes.Actor) (Session, types.Error)
-    GetServicers() []*coreTypes.Actor // the Servicers providing Web3 access to the Application
-    GetFishermen() []*coreTypes.Actor    // the Fishermen monitoring the Servicers
-    GetApplication() *coreTypes.Actor    // the Application consuming Web3 access
-    GetRelayChain() RelayChain        // the identifier of the web3 Relay Chain
-    GetGeoZone() GeoZone              // the geolocation zone where the Application is registered
-    GetSessionHeight() int64          // the block height when the Session started
-}
-```
+# Session Protocol <!-- omit in toc -->
 
-#### Session Creation Flow
+- [Interface](#interface)
+- [Session Creation Flow](#session-creation-flow)
 
-1. Create a session object from the seed data (see #2)
-2. Create a key concatenating and hashing the seed data
-   - `key = Hash(sessionHeight + blockHash + geoZone + relayChain + appPublicKey)`
-3. Get an ordered list of the public keys of servicers who are:
-   - actively staked
-   - staked within geo-zone
-   - staked for relay-chain
-4. Pseudo-insert the session `key` string into the list and find the first actor directly below on the list
-5. Determine a new seedKey with the following formula: ` key = Hash( key + actor1PublicKey )` where `actor1PublicKey` is the key determined in step 4
-6. Repeat steps 4 and 5 until all N servicers are found
-7. Do steps 3 - 6 for Fishermen as well
+_WIP: Run `TODO_SEARCH=utility/session_ make todo*search` to identify all the WIP related to session generation.*
```

Comment:
Markdown ambiguity strikes again; it looks like this line uses `_` to start itilicizing but then uses `*` to end it.